src/nxdoc/nuxeo-server/data-store/data-modeling/schema-fields.md

```diff
@@ -202,6 +202,81 @@ db.default.update(
 )
 ```
 
+## Property Characteristic
+
+A property characteristic represents additional information a property could hold such as secured, deprecated or indexed.
+
+See below the basic contribution for a property characteristic:
+
+```xml
+<component name="my.component.name">
+  <extension target="org.nuxeo.ecm.core.schema.TypeService" point="schema">
+    <property schema="SCHEMA" name="PROPERTY" ... />
+  </extension>
+</component>
+```
+
+`SCHEMA` is the full schema name and `PROPERTY` is the path representing the property, for example:
+- `dublincore` and `created` for the `dc:created` property
+- `files` and `files/*/file/name` for the `files:files/*/file/name` property.
+
+### Secured
+
+When a property is secured, only Administrators can edit them. This prevents regular users to edit sensitive data.
+
+Contribute the xml below to make a property secured:
+
+```xml
+<component name="my.component.name">
+  <extension target="org.nuxeo.ecm.core.schema.TypeService" point="schema">
+    <property schema="SCHEMA" name="PROPERTY" secured="true" />
+  </extension>
+</component>
+```
+
+The following properties are secured by default in Nuxeo Platform:
+- `dc:created`
+- `dc:modified`
+- `dc:creator`
+- `dc:contributors`
+- `dc:lastContributor`
+
+### Indexed
+
+{{#> callout type='info' heading='Only works on MongoDB'}}
+{{/callout}}
+
+Since Nuxeo 2021.7, you can contribute indexes to create a Nuxeo startup. Nuxeo support two types of index: ascending and descending.
+
+Contribute the xml below to make a property indexed:
+
+```xml
+<component name="my.component.name">
+  <extension target="org.nuxeo.ecm.core.schema.TypeService" point="schema">
+    <property schema="SCHEMA" name="PROPERTY" indexOrder="ascending" />
+    <property schema="SCHEMA" name="PROPERTY" indexOrder="descending" />
+  </extension>
+</component>
+```
+
+### Deprecation
+
+You can mark a property as deprecated or removed with or without a fallback (present in the same schema).
+
+A deprecated property still has its declaration in the schema XSD. A warn log will be printed if code tries to read or write this property. The fallback takes the precedence when reading the property, it it is null then the deprecated property is returned. During write operation, both properties receive the new value.
+
+A removed property doesn't have anymore its declaration in the schema XSD. An error log will be printed if code tries to read or write this property. Null will be returned or nothing will be stored if no fallback is defined, otherwise the fallback takes the precedence.
```

Comment:
```suggestion
A removed property doesn't have anymore its declaration in the schema XSD. An error log will be printed if the code tries to read or write this property. Null will be returned and nothing will be stored if no fallback is defined, otherwise, the fallback takes precedence.
```